### PR TITLE
core: plat-ls: Enable caam support for platform LS1046ARDB

### PR DESCRIPTION
**Pull request by priyankasingh01**
Enable CAAM support for platform LS1046ARDB

Signed-off-by: Priyanka Singh <priyanka.singh@nxp.com>
Signed-off-by: Sahil Malhotra <sahil.malhotra@nxp.com>

### CODE REVIEW

**Comment by priyankasingh01** [comment]
@sdininno Please take a look at this

**Comment by pangupta** [comment]
Please merge these changes.

**Comment by jforissier** [comment]
> Acked-off-by: Pankaj Gupta [pankaj.gupta@nxp.com](mailto:pankaj.gupta@nxp.com)

What's this tag? I suppose you mean: `Acked-by: Pankaj Gupta <pankaj.gupta@nxp.com>` right?

File: core/arch/arm/plat-ls/conf.mk [hunk 1]
```diff
@@ -134,3 +136,26 @@ endif
...
+
+# Set
+CFG_CAAM_64BIT ?= y
+
+# Crypto Driver Debug
+CFG_CRYPTO_DRIVER_DEBUG ?= n
+else
+$(call force,CFG_CRYPTO_DRIVER,n)
+$(call force,CFG_WITH_SOFTWARE_PRNG,y)
+
```
Comment (jforissier) [inline core/arch/arm/plat-ls/conf.mk]: No blank line needed here

File: core/arch/arm/plat-ls/platform_config.h [hunk 2]
```diff
@@ -65,8 +65,9 @@
 /*  DUART 1 */
 #define UART0_BASE			0x021C0500
 #define GIC_BASE			0x01400000
-#define GICC_OFFSET			0x2000
-#define GICD_OFFSET			0x1000
+#define GICC_OFFSET			0x20000
+#define GICD_OFFSET			0x10000
```
Comment (jforissier) [inline core/arch/arm/plat-ls/platform_config.h]: These two line look like a fix, do they belong in a separate commit?

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by priyankasingh01** [comment]
> Acked-by: Jerome Forissier <jerome@forissier.org>

Tag updated
